exchanges/kline/types.go

```diff
@@ -1,6 +1,7 @@
 package kline
```

Comment:
Could you rename to `kline_types` for consistency?

Reply:
Yep will get that done shortly and push the change!